health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/Summary.js

```
@@ -0,0 +1,117 @@
+          style={{
+            fontSize: "medium",
+            width: getWidth(data),
+            height: getHeight(data),
+            whiteSpace: "normal",
+          }}
+        >
+          <span style={{ fontWeight: "500", color: "white" }}>{t(`TIP_${data.name}`)}</span>
+          <span style={{ color: "white" }}> {showDate?.[id]?.lastUpdatedTime}</span>
+        </span>
+      </div>
+      <MetricData t={t} data={response?.responseData?.data?.[0]}></MetricData>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)